src/constants/options.ts

```diff
@@ -0,0 +1,16 @@
+import { LiquiditySelection } from '@/lib/utils/balancer/helpers/sor/sorManager';
+import { TradeInterface } from '@/store/modules/app';
+
+export const tradeLiquidityOptions = Object.values(LiquiditySelection)
+  .filter(v => typeof v === 'string')
+  .map(option => ({
+    label: option,
+    value: option
+  }));
+
+export const tradeInterfaceOptions = Object.values(TradeInterface)
+  .filter(v => typeof v === 'string')
+  .map(option => ({
+    label: option,
+    value: option
+  }));
```

Comment:
Since we need these options both in the app settings / trade settings I thought its better to move them into a shared file.